### PR TITLE
update tgi with text-generation-inference:2.1.0

### PR DESCRIPTION
**Pull request by chensuyue**
## Description

update tgi with text-generation-inference:2.1.0 

## Issues

https://github.com/opea-project/GenAIComps/issues/230

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)

## Dependencies

List the newly introduced 3rd party dependency if exists.

## Tests

Describe the tests that you ran to verify your changes.

### CODE REVIEW

File: comps/llms/text-generation/tgi/README.md [hunk 1]
```diff
@@ -114,11 +114,11 @@ curl http://${your_ip}:9000/v1/chat/completions \
...
-| Meta-Llama-3-70B-Instruct | ✓         |
-| Phi-3                     | x         |
+| Model                     | TGI |
+| ------------------------- | --- |
+| Intel/neural-chat-7b-v3-3 | ✓   |
+| Llama-2-7b-chat-hf        | ✓   |
+| Llama-2-70b-chat-hf       | ✓   |
+| Meta-Llama-3-8B-Instruct  | ✓   |
+| Meta-Llama-3-70B-Instruct | ✓   |
+| Phi-3                     | x   |
```
Comment (chensuyue) [inline comps/llms/text-generation/tgi/README.md:124]: Will update the table if test pass.

**Comment by chensuyue** [comment]
@letonghan the test with Phi-3 failed, please check.

**Comment by letonghan** [comment]
> @letonghan the test with Phi-3 failed, please check.

It's mainly caused by the error below. The parameter `--max-input-tokens` should be smaller than the value of `sliding_window=2047`.
image

**Review by lvliang-intel — APPROVED** [review]

File: comps/llms/text-generation/tgi/README.md (hunk 1 above)
Comment (chensuyue) [reply comps/llms/text-generation/tgi/README.md]: Updated

**Pull request merged**